### DISCUSSION TITLE
[@types/alpinejs] Add named exports

### DISCUSSION THREAD

**Discussion by northern-mindset** [DefinitelyTyped/DefinitelyTyped#69404] [Issues with a @types package]
Alpine has got named exports in https://github.com/alpinejs/alpine/pull/3644 (https://github.com/alpinejs/alpine/releases/tag/v3.13.4). I believe this has not been reflected in the types.

When I do

```js
import Alpine from 'alpinejs';
```
I get this eslint error: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-named-as-default.md.

Changing to
```js
import {Alpine} from 'alpinejs';
```
gives me the following TS error:

```
'Alpine' is a type and cannot be imported in JavaScript files. Use 'import("alpinejs").Alpine' in a JSDoc type annotation.ts(18042)
```

Testing on 
- `alpinejs` v3.13.7
- `@types/alpinejs` v3.13.10

[a comment by an automated account is not shown]

**Comment by ekwoka** [comment]
Hmm 🤔 

Strange that I missed that.

It's challenging trying to make types packages for projects with no types

**Reply by northern-mindset** [reply]
Indeed, challenging. So do you think it's possible to adapt?

**Reply by ekwoka** [reply]
Yes. :( Hopefully without breaking too much.

**Comment by ekwoka** [comment]
@anantakrishna So this is not an issue with the types...per se.

Alpine's source code does not do a named export of any kind.

It only has a default export.

However, the current build system, for some reason, adds Alpine as a named export to the distributed script.

I believe it would be unwise to incorporate a named export into the types, as the source does not actually provide any guarantee of that being included in the future.

I'd advise you to also not use the named export for the same reason as above. Such a change would be unlikely to be marked as a breaking change, and is more an unintended side effect.
